pkg/skaffold/deploy/helm/deploy.go

```diff
@@ -89,38 +89,40 @@ type Deployer struct {
 	bV semver.Version
 }
 
-// NewDeployer returns a configured Deployer
-func NewDeployer(cfg kubectl.Config, labels map[string]string) *Deployer {
+// NewDeployer returns a configured Deployer if current version of helm is greater than 3.0.0
+func NewDeployer(cfg kubectl.Config, labels map[string]string) (*Deployer, error) {
+	hv, err := binVer()
+	if err != nil {
+		return nil, fmt.Errorf(versionErrorString, err)
+	}
+
+	if hv.LT(helm3Version) {
+		return nil, fmt.Errorf("skaffold requires Helm version 3.0.0-beta.0 or greater")
```

Comment:
I wonder if we need to support 3.0.0 betas :-)

Reply:
i changed the error message to say -> 3.0.0 or greater. 
Regarding supporting `beta.0`, i feel it could be a big change if we stop supporting in this PR. Also needs some investigation. I created a bug here #5081